tools/dotnet-linker/LinkerConfiguration.cs

```diff
@@ -263,12 +263,47 @@ public static LinkerConfiguration GetInstance (LinkContext context)
 				case "PlatformAssembly":
 					PlatformAssembly = Path.GetFileNameWithoutExtension (value);
 					break;
+				case "ReferenceNativeSymbol": {
+					(string symbolType, string symbolMode, string symbol) = SplitString3 (value, ':');
+					var mode = SymbolMode.Default;
+					switch (symbolMode) {
+					case "Ignore":
+						mode = SymbolMode.Ignore;
+						break;
+					case "Default":
```

Comment:
Naming is hard! What is `Default`? :) maybe add a third value called `Include` and `Include` == `Default`? or just rename `Default` to `Include`? It is just not clear what `Default` means from just `Default` and `Ignore`

Reply:
@dalexsoto I could just remove `"Default"` - then there would be two options: `"Ignore"` and `""` (empty string - which would really be "not ignore"). Maybe that makes more sense?

Reply:
@dalexsoto I removed `Default` to simplify.